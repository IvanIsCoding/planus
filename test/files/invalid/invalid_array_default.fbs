table Foo {
  x: [uint8: 3] = [0, 1, 2, 3];
}