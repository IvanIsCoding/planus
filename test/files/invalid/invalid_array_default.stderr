[0m[1m[38;5;9merror[0m[1m: Unsupported type[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_array_default.fbs:2:6
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [0m[31m[uint8: 3][0m = [0, 1, 2, 3];
  [0m[34mâ”‚[0m      [0m[31m^^^^^^^^^^[0m [0m[31mArrays in tables are not supported[0m

[0m[1m[38;5;9merror[0m[1m: Array literal does not have the correct length[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_array_default.fbs:2:19
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [uint8: 3] = [0m[31m[0, 1, 2, 3][0m;
  [0m[34mâ”‚[0m                   [0m[31m^^^^^^^^^^^^[0m

Error: Bailing because of errors
